Logger/Book.cs

```diff
@@ -0,0 +1,52 @@
+﻿using System;
+
+namespace Logger;
+
+public record Book : IEntity
+{
+    // implemented implicitly since Name is a frequently-accessed member
+    public string Name { get; set; }
+
+    //TODO: explain why this is implemeted implicitly
+    public Guid Id { get; init; }
+
+    private FullName _author;
+        public string Author
+    {
+        get => _author.ToString();
+        set => _author = new(value);
+    }
+    public int Isbn { get; init; }
+
+    public Book(string name, string author, int isbn, Guid id)
+    {
+        Id = id;
+        Name = name;
+        _author = new(author);
```

Comment:
Remember, we access fields through their properties
```suggestion
        Author = new(author);
```